src/mame/namco/namcos21_3d.cpp

```diff
@@ -70,6 +71,7 @@ void namcos21_3d_device::copy_visible_poly_framebuffer(bitmap_ind16 &bitmap, con
 		uint16_t *const dest = &bitmap.pix(sy);
 		uint16_t const *const pPen = m_mpPolyFrameBufferPens2.get() + m_poly_frame_width * sy;
 		uint16_t const *const pZ = m_mpPolyFrameBufferZ2.get() + m_poly_frame_width * sy;
+
 		for (int sx = clip.left(); sx <= clip.right(); sx++)
 		{
```

Comment:
Another blank line that doesn’t really make the code more readable.

Reply:
Done.